packages/react-core/src/components/FormSelect/__tests__/__snapshots__/FormSelect.test.tsx.snap

```diff
@@ -1,311 +1,23 @@
-    data-ouia-component-id="OUIA-Generated-FormSelect-default-1"
-    data-ouia-component-type="PF4/FormSelect"
-    data-ouia-safe="true"
-  >
-    <option
-      class=""
-      disabled=""
-      value="please choose"
-    >
-      Please Choose
-    </option>
-    <option
-      class=""
-      value="mr"
-    >
-      Mr
-    </option>
-    <option
-      class=""
-      value="miss"
-    >
-      Miss
-    </option>
-    <option
-      class=""
-      value="mrs"
-    >
-      Mrs
-    </option>
-    <option
-      class=""
-      value="ms"
-    >
-      Ms
-    </option>
-    <option
-      class=""
-      value="dr"
-    >
-      Dr
-    </option>
-    <option
-      class=""
-      disabled=""
-      value="other"
-    >
-      Other
-    </option>
-  </select>
-</div>
-`;
+exports[`FormSelect Simple FormSelect input 1`] = `"<select aria-label=\\"simple FormSelect\\" class=\\"pf-c-form-control\\" aria-invalid=\\"false\\" data-ouia-component-type=\\"PF4/FormSelect\\" data-ouia-safe=\\"true\\" data-ouia-component-id=\\"OUIA-Generated-FormSelect-default-1\\"><option class=\\"\\" value=\\"please choose\\" disabled=\\"\\">Please Choose</option><option class=\\"\\" value=\\"mr\\">Mr</option><option class=\\"\\" value=\\"miss\\">Miss</option><option class=\\"\\" value=\\"mrs\\">Mrs</option><option class=\\"\\" value=\\"ms\\">Ms</option><option class=\\"\\" value=\\"dr\\">Dr</option><option class=\\"\\" value=\\"other\\" disabled=\\"\\">Other</option></select>"`;
 
-exports[`invalid FormSelect input 1`] = `
-<div>
-  <select
-    aria-invalid="true"
-    aria-label="invalid FormSelect"
-    class="pf-c-form-control"
-    data-ouia-component-id="OUIA-Generated-FormSelect-error-1"
-    data-ouia-component-type="PF4/FormSelect"
-    data-ouia-safe="true"
-  >
-    <option
-      class=""
-      value="mr"
-    >
-      Mr
-    </option>
-  </select>
-</div>
-`;
+exports[`FormSelect invalid FormSelect input 1`] = `"<select aria-label=\\"invalid FormSelect\\" class=\\"pf-c-form-control\\" aria-invalid=\\"true\\" data-ouia-component-type=\\"PF4/FormSelect\\" data-ouia-safe=\\"true\\" data-ouia-component-id=\\"OUIA-Generated-FormSelect-error-1\\"><option class=\\"\\" value=\\"mr\\">Mr</option></select>"`;
 
-exports[`required FormSelect input 1`] = `
-<div>
-  <select
-    aria-invalid="false"
-    aria-label="required FormSelect"
-    class="pf-c-form-control"
-    data-ouia-component-id="OUIA-Generated-FormSelect-default-7"
-    data-ouia-component-type="PF4/FormSelect"
-    data-ouia-safe="true"
-  >
-    <option
-      class=""
-      value="mr"
-    >
-      Mr
-    </option>
-  </select>
-</div>
-`;
+exports[`FormSelect required FormSelect input 1`] = `"<select aria-label=\\"required FormSelect\\" class=\\"pf-c-form-control\\" aria-invalid=\\"false\\" data-ouia-component-type=\\"PF4/FormSelect\\" data-ouia-safe=\\"true\\" data-ouia-component-id=\\"OUIA-Generated-FormSelect-default-7\\"><option class=\\"\\" value=\\"mr\\">Mr</option></select>"`;
```

Comment:
The problem was the prop was wrong and is a perfect example of a false-positive snapshot test. The prop was meant to be `isRequired` having inspected the component further. 

I've updated the test to verify the `required` attribute is present when the prop is specified instead of taking a snapshot.